### PR TITLE
Update metadata getter

### PR DESCRIPTION
**Pull request by adamrp**
Now it accepts either a file as input, or a comma-separated list of barcodes, or both, just for convenience

### CODE REVIEW

File: python_code/scripts/ag_get_metadata.py [hunk 1]
```diff
@@ -129,11 +136,22 @@ def main():
         'NITROMIDAZOLE', 'PENICILLIN', 'SULFA_DRUG', 'CEPHALOSPORIN'
     ]
 
+    all_barcodes = []
+
+    if input_fp is None and barcodes is None:
+        raise IOError("Must supply either -i or -b")
```
Comment (ElDeveloper) [inline python_code/scripts/ag_get_metadata.py]: Can you use `option_parser.error`?

File: python_code/scripts/ag_get_metadata.py [hunk 2]
```diff
@@ -51,7 +49,15 @@
 
 script_info['optional_options'] = [
     make_option('-H', '--omit_headers', action='store_true',
-                help="Do not print column headers as the first line")
+                help="Do not print column headers as the first line"),
+    make_option('-i', '--input_barcodes_file', type='existing_filepath',
+                help="The input file containing barcodes, one per line. "
+                "At least one of -b or -i must be supplied.  If both are "
```
Comment (ElDeveloper) [inline python_code/scripts/ag_get_metadata.py]: I don't think this information should be in the help text for the options but instead in the script-level description/help. What do you think?

File: python_code/scripts/ag_get_metadata.py (hunk 1 above)
Comment (adamrp) [inline python_code/scripts/ag_get_metadata.py]: Ah yeah, I meant to do that! I couldn't remember the error type at first. Changing!

File: python_code/scripts/ag_get_metadata.py (hunk 2 above)
Comment (adamrp) [inline python_code/scripts/ag_get_metadata.py]: Makes sense I guess, I'll do that

**Comment by ElDeveloper** [comment]
Thanks!

**Comment by teravest** [comment]
Looks good

**Comment by ElDeveloper** [comment]
Thanks @adamrp!!

Yoshiki Vázquez-Baeza

**Pull request merged**